pillar.example

```diff
@@ -44,6 +44,20 @@ systemd:
       Install:
         WantedBy: multi-user.target
 
+  ## resolved
+  resolved:
+    config:
+      DNS: 1.1.1.1 1.0.0.1 2606:4700:4700::1111 2606:4700:4700::1001
+      FallbackDNS: 8.8.8.8 8.8.4.4 2001:4860:4860::8888 2001:4860:4860::8844
+      Domains: '""'
+      LLMNR: 'True'
+      MulticastDNS: 'True'
+      DNSSEC: 'allow-downgrade'
+      DNSOverTLS: 'False'
+      Cache: 'True'
+      DNSStubListener: 'udp'
```

Comment:
Likewise, [this comment](https://github.com/saltstack-formulas/systemd-formula/pull/28/files?file-filters%5B%5D=.example&file-filters%5B%5D=.sls&file-filters%5B%5D=.yaml#r268825377).